felix/bpf/proxy/syncer.go

```diff
@@ -707,8 +707,12 @@ func (s *Syncer) updateService(skey svcKey, sinfo k8sp.ServicePort, id uint32, e
 		if !ep.GetIsLocal() {
 			continue
 		}
-		if err := s.writeSvcBackend(id, uint32(cnt), ep); err != nil {
```

Comment:
Update code to prevent terminating pods from being added to the bpf maps as outlined above @tomastigera